data_steward/utils/pipeline_logging.py

```diff
@@ -23,7 +23,7 @@
 
 LOG_FORMAT = '%(asctime)s - %(levelname)s - %(name)s - %(message)s'
 LOG_DATEFMT = '%Y-%m-%d %H:%M:%S'
-FILENAME_FMT = '%Y%m%d'
+FILENAME_FMT = '%Y%m%d-' + os.getenv('GOOGLE_CLOUD_PROJECT', 'UNDEFINED_GOOGLE_PROJECT')
```

Comment:
A few notes:

1. We generally use f-strings where possible.  I think this is newish in python 3.X.
2. We already get the project_id via an import variable in https://github.com/all-of-us/curation/blob/develop/data_steward/app_identity.py#L6

With both of those considerations (but I don't think f-strings apply in this special situation), we should probably be doing:
```suggestion
FILENAME_FMT = '%Y%m%d-' + get_application_id()
```
and `from app_identity import get_application_id`.

Reply:
Also, the linting check will fail if the PR title doesn't conform to https://github.com/all-of-us/curation/blob/develop/tools/validate_pr_title.sh .  Ticket number should be in brackets followed by a space.